lib/screens/v2/candidate_generator/solari_large.ex

```diff
@@ -26,7 +26,10 @@ defmodule Screens.V2.CandidateGenerator.SolariLarge do
   end
 
   defp header_instances(config, now) do
-    %Screen{app_params: %Solari{station_name: header_text}} = config
-    [%NormalHeader{screen: config, text: header_text, time: now}]
+    %Screen{
+      app_params: %SolariLarge{header: %Header{type: :current_stop_name, stop_name: stop_name}}
```

Comment:
Hmm. Should assertions like `type: :current_stop_name` happen here? I can't think of a better way to do it if we want to have one shared header config but only support one variant of it for Solari screens, but it feels iffy to have these subtle constraints on the config spread about.